static/scss/_button.scss

```diff
@@ -1,35 +1,42 @@
 .mm-button {
   background-color: #F1F1F1;
   border-radius: 2px;
-  padding: 6px 12px;
-  border: 1px solid #d3d3d3;
+  border: 0;
+  padding: 12px 35px;
   align-items: center;
   display: flex;
   flex-direction: row;
   vertical-align: middle;
   white-space: nowrap;
+  text-transform: uppercase;
+  font-size: 16px;
+  font-weight: 400;
+  margin: 0 15px 0 0;
+  cursor: pointer;
+
+  &.main-action {
+    background-color: $mm-brand-bg;
+    color: white;
+  }
+
+  &.minor-action {
+    font-size: 13px;
+    padding: 9px 19px;
+    border: 1px solid #e6e6e6;
+    background-image: -ms-linear-gradient(top, #f8f8f8   0%, #F1F1F1   100%);
+    background-image: -moz-linear-gradient(top, #f8f8f8   0%, #F1F1F1   100%);
+    background-image: -o-linear-gradient(top, #f8f8f8   0%, #F1F1F1   100%);
+    background-image: -webkit-gradient(linear, left top, left bottom, color-stop(0, #f8f8f8  ), color-stop(100, #F1F1F1  ));
+    background-image: -webkit-linear-gradient(top, #f8f8f8   0%, #F1F1F1   100%);
+    background-image: linear-gradient(to bottom, #f8f8f8   0%, #F1F1F1   100%);
+  }
 }
 
 .mm-minor-action {
-  color: $mm-brand-blue;
+  color: $link-text;
   font-size: 16px;
   font-weight: 600;
-  cursor: pointer
-}
-
-.mm-button-action {
```

Comment:
It looks like this is still used in two places, in `ProgressWidget` and `CourseAction`. If we're fine with how they look now we should remove the references to `mm-button-action` there